src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const MIDDLE_TERM = 3;
+
+  const MIDDLE_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+
+  const LONG_TERM_DISCOUNT = 50;
+  let totalAmount = days * PRICE_PER_DAY;
```

Comment:
Consider using `const` instead of `let` for `totalAmount` since its value is not reassigned after its initial calculation. This follows the checklist item to prefer `const` over `let` where possible to avoid unintentional changes.